tracee-rules/signature.go

```diff
@@ -97,7 +98,11 @@ func findRegoSigs(dir string) ([]types.Signature, error) {
 		}
 		sig, err := regosig.NewRegoSignature(string(regoCode), regoHelpersCode)
 		if err != nil {
-			log.Printf("error creating rego signature with: %s: %v ", regoCode[0:20], err)
+			newlineOffset := bytes.Index(regoCode, []byte("\n"))
+			if newlineOffset == -1 {
+				newlineOffset = len(regoCode)
```

Comment:
So I think it doesn't really matter, but in this case it'll print the whole rego code. Maybe we limit it at 20 like above? Perhaps just change this to `len(regoCode)%20`? or some other arbitrary (but reasonable) limit number. WDYT?